l10ninsp/slave.py

```diff
@@ -123,6 +132,19 @@ def finished(self, *args):
       rc = FAILURE
     self.sendStatus({'rc': rc})
 
+def compareDirs(path1, path2):
+    cljsPath = '/home/elmo/elmo/slave-ball/vendor-local/compare-locales.js'
+    # nodejs ./bin/compare-dirs.js --data=json ~/elmo/stage/workdir/a
+    process = subprocess.Popen(
+      ["/usr/bin/nodejs",
```

Comment:
That's what the binary is called in Debian/Ubuntu: https://lists.debian.org/debian-devel-announce/2012/07/msg00002.html.  A common solution is setting a symlinks according to the user's preference.  Not sure if we can detect that somehow here.